### PR TITLE
[Enhancement](checker) Enhance inverted checker

### PR DESCRIPTION
**Pull request by Yukang-Lian**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

- Behavior changed:

    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?

    - [ ] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: cloud/src/common/bvars.cpp [hunk 1]
```diff
@@ -191,3 +191,6 @@ BvarStatusWithTag<long> g_bvar_checker_check_cost_s("checker", "check_cost_secon
 BvarStatusWithTag<long> g_bvar_checker_enqueue_cost_s("checker", "enqueue_cost_seconds");
 BvarStatusWithTag<long> g_bvar_checker_last_success_time_ms("checker", "last_success_time_ms");
 BvarStatusWithTag<long> g_bvar_checker_instance_volume("checker", "instance_volume");
+BvarStatusWithTag<long> g_bvar_inverted_checker_num_scanned("checker", "num_inverted_scanned");
```
Comment (gavinchou) [inline cloud/src/common/bvars.cpp:194]: is it number of "object/file" or "kv"?

File: cloud/src/recycler/checker.cpp [hunk 2]
```diff
@@ -170,7 +170,7 @@ int Checker::start() {
             g_bvar_checker_enqueue_cost_s.put(instance_id, ctime_ms / 1000 - enqueue_time_s);
             ret = checker->do_check();
             if (config::enable_inverted_check) {
-                if (checker->do_inverted_check() != 0) ret = -1;
+                ret = checker->do_inverted_check();
```
Comment (gavinchou) [inline cloud/src/recycler/checker.cpp:173]: seems logic differs from the original

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run cloud_p0

**Review by luwei16 — APPROVED** [review]

**Comment by Yukang-Lian** [comment]
run beut

**Review by gavinchou — DISMISSED** [review]

**Comment by Yukang-Lian** [comment]
run beut

**Comment by Yukang-Lian** [comment]
run beut

**Comment by Yukang-Lian** [comment]
run beut

**Comment by Yukang-Lian** [comment]
run beut

**Comment by Yukang-Lian** [comment]
run beut

**Comment by gavinchou** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run cloud_p0

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Pull request merged**